packages/modal/storybook/Modal.js

```diff
@@ -7,6 +7,11 @@ import styled from 'styled-components'
 const StyledModal = styled(Modal)`
   height: ${props => props.height};
 `
+const CUSTOM_ANIMATION = transitionstate => `
+  transform: translateY(-122%);
+  transition: transform 0.3s linear;
+  ${transitionstate === 'entered' ? `transform: translateY(0%);` : ''}
```

Comment:
Consistent camel case for variable name?

```suggestion
  ${transitionState === 'entered' ? `transform: translateY(0%);` : ''}
```